apps/frontend/src/app/pages/company/create-profile/create-profile.component.html

```diff
@@ -0,0 +1,366 @@
+<form class="container px-5 mx-auto min-h-screen" [formGroup]="form">
+  <div class="md:flex gap-36 pt-8">
+    @if(step === 1){
+    <div class="w-full">
+      <div class="md:flex w-full gap-8">
+        <div
+          class="container flex justify-end bg-neutral border-2 rounded-3xl border-base-300 w-48 h-48"
+        >
+          @if(!profilePhoto){
+          <div class="rounded-3xl bg-neutral object-cover w-48 h-46"></div>
+          } @if(profilePhoto){
+          <img
+            class="rounded-3xl object-cover w-48 h-46"
+            [src]="profilePhoto"
+            alt="profile company"
+          />
+          }
+          <div
+            class="absolute rounded-3xl pt-40 pr-3 w-9 h-9 flex items-center justify-center cursor-pointer"
+            (click)="profilePhoto.click()"
+          >
+            <input
+              #profilePhoto
+              class="hidden"
+              type="file"
+              accept="image/*"
+              (change)="onFileChange($event)"
+            />
+            <i
+              class="bi bi-pencil text-sm text-primary shadow bg-neutral py-2 px-3 rounded-full"
+              aria-hidden="true"
+            ></i>
+          </div>
+        </div>
+        <div class="w-full">
+          <div class="w-full">
+            <wagademy-form-field
+              [control]="form.controls.companyName"
+              label="Company name"
+            >
+              <wagademy-input placeholder="Company name"></wagademy-input>
+            </wagademy-form-field>
+            <div class="pt-4">
+              <wagademy-form-field
+                [control]="form.controls.areaOfExpertise"
+                label="Area of expertise"
+              >
+                <wagademy-input
+                  placeholder="Area of expertise"
+                ></wagademy-input>
+              </wagademy-form-field>
+            </div>
+            <div class="pb-6">
+              <wagademy-form-field
+                [control]="form.controls.description"
+                label="Description"
```

Comment:
Here instead of description use About.
I know it was in figma, but in the backend i did the name about and also in agreement with Luiz to change the name to "About"